web_register_login/models/res_users.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2020 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResUsers(models.Model):
+
+    _inherit = "res.users"
+
+    dont_register_login = fields.Boolean()
```

Comment:
`dont_register_login` or `no_register_login`?

Reply:
Both LGTM, we can keep it like `dont_register_login` to make it easier to read...